File: Assignment_25/program4.c
/////////////////////////////////////////////////////////////////////////
//
// Name :          program3.c
// Discription :   Display digits from string
// Input :         String
// Output :        string
// Author :        Shivam Rajendra Kale
// Date :          08/06/2025
//
//////////////////////////////////////////////////////////////////////////

/*
   Start
      Accept String from user 
         Display digits from string
    Stop         
*/

#include<stdio.h>

void DisplayDigit(char *str)
{
   int iDisplacement = 0;

   if(NULL == str)
   {
      return ;
   }

   while(*str != '\0')
   {
      if((*str >= '1') && (*str <= '9'))
      {
         printf("%c",*str);   
      }
      str++;
   }

}

int main()
{
   char Arr[20] = {'\0'};

   printf("Enter string :\n");
   scanf("%[^'\n']s",Arr);

   DisplayDigit(Arr); 

   return 0;

}

File: Assignment_55/program4.java
//////////////////////////////////////////////////////////////////////////
//
// Name :          program4.java
// Discription :   use to Display Diff of small & capital characters
// Input :         Integer
// Output :        Integer
// Author :        Shivam Rajendra Kale
// Date :          19/10/2025
//
//////////////////////////////////////////////////////////////////////////

import java.util.*;

class MyArray
{
    public int Search(char Arr[])
    {
        int i = 0, Count1 = 0, Count2 = 0;
    
        for(i = 0; i < Arr.length; i++)
        {
            if(Arr[i] >= 'A' && Arr[i] <= 'Z')
            {
                Count1++;
            }
            if(Arr[i] >= 'a' && Arr[i] <= 'z')
            {
                Count2++;
            }
        }

        return Count1 - Count2;
    }
}

class program4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        int i = 0, iRet = 0;

        System.out.println("Enter Array of Characters : ");
        String str = sobj.nextLine();

        char Arr[] = str.toCharArray();

        MyArray mobj = new MyArray();

        iRet = mobj.Search(Arr);
        System.out.println("Frequence of is : "+iRet);

        sobj.close();
    }
}